docs/source/dev_guide/release.rst

```diff
@@ -63,7 +63,12 @@ Bumping the Version
 
 .. _github-release:
 
-Releasing on GitHub
+Releasing on GitHub: release candidates
+-------------------
+
+1. Create a tag for the release candidate at https://github.com/E3SM-Project/zppy/tags.
```

Comment:
I don't see a way to create a tag without creating a release. https://stackoverflow.com/questions/64561146/how-do-i-create-a-tag-in-github-without-creating-a-release suggests creating a release and then deleting it. Is it something to do on the command-line side (https://git-scm.com/book/en/v2/Git-Basics-Tagging)?

Reply:
We currently use the pre-release feature for release-candidates (https://github.com/E3SM-Project/zppy/releases/tag/v2.2.0rc4 for instance). I do like that as it allows me to write up a summary of what's changed between RCs. That said, Xylar does have a point about it confusing people; it's annoying to sort through multiple RCs to get to actual releases.